src/gentropy/ld_index.py

```diff
@@ -57,6 +59,7 @@ def __init__(
             GnomADLDMatrix(
                 ld_matrix_template=ld_matrix_template,
                 ld_index_raw_template=ld_index_raw_template,
+                grch37_to_grch38_chain_path=grch37_to_grch38_chain_path,
```

Comment:
This is OK for now, but we need to phase it out in favour of the liftover table provided by Hail, which we are already using to extract the LD matrix for a locus

Reply:
I will look over this one this week, but lets stick to the above changes for now. I have made it into an issue, feel free to modify it. https://github.com/opentargets/issues/issues/3342

Reply:
Cool, thank you!